src/main/java/com/zaxxer/sansorm/internal/Introspected.java

```diff
@@ -43,11 +44,10 @@
  */
 public class Introspected
 {
-   private Class<?> clazz;
+   private final Class<?> clazz;
+   private final Map<String, FieldColumnInfo> columnToField = new LinkedHashMap<>();
```

Comment:
i'm not sure why are you using static/instance initializers instead of simple inlined ones. any reason for that?

Reply:
We had generally preferred no inline initialization because it meant that if we had any logic that needed to be performed as part of initialization then the code would still be in the same place. So if as a rule all initialization is in the initializer or constructor, then you would only have to look in a single block of code to see it all.

When you have a single initializer like this, it might not seem useful, but by sticking with the same style across an entire code base, it reduces the possible version control diffs when you do introduce some sort of logic that needs to be run.

It also used to be a pain in the debugger when there were inline initializers. These days that doesn't seem to be a problem anymore.

This idea was from years ago, I personally don't mind the inline initializer now. (especially since most people don't even know about the instance initializer syntax)

I suspect @brettwooldridge also doesn't mind. But I can't always speak for him.

Reply:
ok so what should i do? revert or leave it as is?